test/wdio.conf.mjs

```diff
@@ -370,6 +360,7 @@ function buildConfig() {
           'osVersion': 'Mavericks',
           'seleniumVersion': '3.14.0',
         },
+        'rav:send:skipCrossDomainTest': true,
```

Comment:
The cross domain test was being flaky for this very old version of Safari, so I've disabled it for now (similar to what we've done for IE browsers). 🤷